# Translations template for ceilometer.
# Copyright (C) 2015 ORGANIZATION
# This file is distributed under the same license as the ceilometer project.
#
# Translators:
# Andreas Jaeger <jaegerandi@gmail.com>, 2016. #zanata
msgid ""
msgstr ""
"Project-Id-Version: ceilometer 6.0.1.dev170\n"
"Report-Msgid-Bugs-To: https://bugs.launchpad.net/openstack-i18n/\n"
"POT-Creation-Date: 2016-06-07 17:37+0000\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"PO-Revision-Date: 2016-03-10 08:01+0000\n"
"Last-Translator: Grigory Mokhin <mokhin@gmail.com>\n"
"Language: ru\n"
"Plural-Forms: nplurals=4; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n"
"%10<=4 && (n%100<12 || n%100>14) ? 1 : n%10==0 || (n%10>=5 && n%10<=9) || (n"
"%100>=11 && n%100<=14)? 2 : 3);\n"
"Generated-By: Babel 2.0\n"
"X-Generator: Zanata 3.7.3\n"
"Language-Team: Russian\n"

#, python-format
msgid "%(entity)s %(id)s Not Found"
msgstr "%(entity)s %(id)s не найден"

#, python-format
msgid "Arithmetic transformer must use at least one meter in expression '%s'"
msgstr ""
"Арифметический преобразователь должен использовать хотя бы один счетчик в "
"выражении %s'"

#, python-format
msgid "Cannot create table %(table_name)s   it already exists. Ignoring error"
msgstr ""
"Не удалось создать таблицу %(table_name)s: уже существует. Игнорирование "
"ошибки"

#, python-format
msgid "Continue after error from %(name)s: %(error)s"
msgstr "Продолжить после ошибки с %(name)s: %(error)s"

#, python-format
msgid "Could not connect slave host: %s "
msgstr "Не удалось подключиться к подчиненному хосту: %s "

#, python-format
msgid "Could not connect to XenAPI: %s"
msgstr "Не удалось подключиться к XenAPI: %s"

#, python-format
msgid "Could not get CPU Util for %(id)s: %(e)s"
msgstr ""
"Не удалось получить информацию об использовании процессора для %(id)s: %(e)s"

#, python-format
msgid "Could not get Memory Usage for %(id)s: %(e)s"
msgstr ""
"Не удалось получить информацию об использовании памяти для %(id)s: %(e)s"

#, python-format
msgid "Could not get VM %s CPU Utilization"
msgstr ""
"Не удалось получить информацию об использовании CPU для виртуальной машины %s"

#, python-format
msgid "Couldn't obtain IP address of instance %s"
msgstr "Не удалось получить IP-адрес экземпляра %s"

#, python-format
msgid "Dropping Notification %(type)s (uuid:%(msgid)s)"
msgstr "Удаление уведомления %(type)s (uuid:%(msgid)s)"

#, python-format
msgid ""
"Error from libvirt while looking up instance <name=%(name)s, id=%(id)s>: "
"[Error Code %(error_code)s] %(ex)s"
msgstr ""
"Возникла ошибка в libvirt при поиске экземпляра <имя=%(name)s, ИД=%(id)s>: "
"[Код ошибки: %(error_code)s] %(ex)s"

#, python-format
msgid "Error parsing HTTP response: %s"
msgstr "Ошибка анализа ответа HTTP: %s"

msgid "Error stopping pollster."
msgstr "Ошибка остановки опрашивающего объекта."

msgid "Event"
msgstr "Событие"

msgid "Expression evaluated to a NaN value!"
msgstr "Результат вычисления выражения - значение NaN!"

#, python-format
msgid "Failed to import extension for %(name)s: %(error)s"
msgstr "Не удалось импортировать расширение для %(name)s: %(error)s"

#, python-format
msgid ""
"Failed to inspect data of instance <name=%(name)s, id=%(id)s>, domain state "
"is SHUTOFF."
msgstr ""
"Не удалось проверить данные экземпляра <имя=%(name)s, ИД=%(id)s>, состояние "
"домена - SHUTOFF."

#, python-format
msgid ""
"Failed to inspect memory usage of %(instance_uuid)s, can not get info from "
"libvirt: %(error)s"
msgstr ""
"Не удалось проверить использование памяти экземпляром %(instance_uuid)s, не "
"удалось получить информацию от libvirt: %(error)s"

#, python-format
msgid ""
"Failed to inspect memory usage of instance <name=%(name)s, id=%(id)s>, can "
"not get info from libvirt."
msgstr ""
"Не удалось проверить использование памяти экземпляром <имя=%(name)s, ИД="
"%(id)s>, не удалось получить информацию от libvirt."

#, python-format
msgid "Failed to load any notification handlers for %s"
msgstr "Не удалось загрузить обработчики уведомлений для %s"

#, python-format
msgid "Failed to parse the timestamp value %s"
msgstr "Ошибка анализа значения времени %s"

#, python-format
msgid "Failed to publish %d datapoints, dropping them"
msgstr "Не удалось опубликовать %d точек данных, выполняется их удаление"

#, python-format
msgid "Failed to publish %d datapoints, queue them"
msgstr "Не удалось опубликовать %d точек данных, создайте для них очередь"

#, python-format
msgid "Filter expression not valid: %s"
msgstr "Недопустимое выражение фильтра: %s"

#, python-format
msgid "Ignoring instance %(name)s (%(instance_id)s) : %(error)s"
msgstr "Игнорирование экземпляра %(name)s (%(instance_id)s) : %(error)s"

#, python-format
msgid "Ignoring instance %(name)s: %(error)s"
msgstr "Игнорирование экземпляра %(name)s: %(error)s"

#, python-format
msgid "Ignoring loadbalancer %(loadbalancer_id)s"
msgstr "Балансировщик нагрузки %(loadbalancer_id)s игнорируется"

#, python-format
msgid "Ignoring pool %(pool_id)s"
msgstr "Пул %(pool_id)s игнорируется"

#, python-format
msgid ""
"Invalid YAML syntax in Definitions file %(file)s at line: %(line)s, column: "
"%(column)s."
msgstr ""
"Недопустимый синтаксис YAML в файле определений %(file)s; строка: %(line)s, "
"столбец: %(column)s."

#, python-format
msgid "Invalid period %(period)s: %(err)s"
msgstr "Недопустимый интервал %(period)s: %(err)s"

#, python-format
msgid "Invalid trait type '%(type)s' for trait %(trait)s"
msgstr "Недопустимый тип особенности %(type)s для особенности %(trait)s"

msgid "Limit must be positive"
msgstr "Ограничение должно быть положительным"

#, python-format
msgid "More than one event with id %s returned from storage driver"
msgstr "Из драйвера хранилища возвращено несколько событий с ИД %s"

#, python-format
msgid "Multiple VM %s found in XenServer"
msgstr "Найдено несколько виртуальных машин %s в XenServer"

msgid "Must specify connection_url, and connection_password to use"
msgstr "Необходимо указать connection_url и connection_password"

#, python-format
msgid "No plugin named %(plugin)s available for %(name)s"
msgstr "Нет доступного модуля %(plugin)s для %(name)s"

msgid "Node Manager init failed"
msgstr "Сбой инициализации администратора узлов"

#, python-format
msgid "Not Authorized to access %(aspect)s %(id)s"
msgstr "Нет прав доступа к %(aspect)s %(id)s"

#, python-format
msgid "OpenDaylitght API returned %(status)s %(reason)s"
msgstr "Функция API OpenDaylight вернула %(status)s %(reason)s"

#, python-format
msgid "Opencontrail API returned %(status)s %(reason)s"
msgstr "API Opencontrail возвратил %(status)s %(reason)s"

#, python-format
msgid ""
"Operator %(operator)s is not supported. Only equality operator is available "
"for field %(field)s"
msgstr ""
"Оператор %(operator)s не поддерживается. Для поля %(field)s возможен только "
"оператор равенства"

#, python-format
msgid ""
"Operator %(operator)s is not supported. The supported operators are: "
"%(supported)s"
msgstr ""
"Оператор %(operator)s не поддерживается. Поддерживаемые операторы: "
"%(supported)s"

#, python-format
msgid "Order-by expression not valid: %s"
msgstr "Недопустимое выражение сортировки: %s"

#, python-format
msgid ""
"Parse error in JSONPath specification '%(jsonpath)s' for %(name)s: %(err)s"
msgstr ""
"Ошибка анализа спецификации JSONPath %(jsonpath)s для %(name)s: %(err)s"

msgid "Period must be positive."
msgstr "Период должен быть положительным."

#, python-format
msgid "Pipeline %(pipeline)s: %(status)s after error from publisher %(pub)s"
msgstr "Конвейер %(pipeline)s: %(status)s после ошибки от публикатора %(pub)s"

#, python-format
msgid "Pipeline %(pipeline)s: Continue after error from publisher %(pub)s"
msgstr "Конвейер %(pipeline)s: Продолжение после ошибки из публикатора %(pub)s"

#, python-format
msgid "Pipeline %(pipeline)s: Error flushing transformer %(trans)s"
msgstr "Конвейер %(pipeline)s: Ошибка выгрузки преобразователя %(trans)s"

#, python-format
msgid ""
"Pipeline %(pipeline)s: Exit after error from transformer %(trans)s for "
"%(smp)s"
msgstr ""
"Конвейер %(pipeline)s: Выход после ошибки из преобразователя %(trans)s для "
"%(smp)s"

#, python-format
msgid "Plugin specified, but no plugin name supplied for %s"
msgstr "Указан модуль, но не передано имя модуля для %s"

#, python-format
msgid "Polling %(mtr)s sensor failed for %(cnt)s times!"
msgstr "%(cnt)s-кратный сбой датчика опроса %(mtr)s!"

#, python-format
msgid "Polling %(name)s failed for %(cnt)s times!"
msgstr "Опрос %(name)s не удалось выполнить %(cnt)s раз."

#, python-format
msgid "Pollster for %s is disabled!"
msgstr "Опрашивающий объект для %s выключен!"

#, python-format
msgid "Prevent pollster %(name)s for polling source %(source)s anymore!"
msgstr ""
"Сделайте так, чтобы опрашивающий объект %(name)s больше не опрашивал "
"источник %(source)s!"

#, python-format
msgid ""
"Publisher max local_queue length is exceeded, dropping %d oldest samples"
msgstr ""
"Превышена максимальная длина local_queue публикатора, удаление %d самых "
"старых образцов"

#, python-format
msgid "Publishing policy is unknown (%s) force to default"
msgstr "Стратегия публикации неизвестна (%s). По умолчанию принудительная"

#, python-format
msgid "RGW AdminOps API returned %(status)s %(reason)s"
msgstr "Функция API RGW AdminOps вернула %(status)s %(reason)s"

msgid "Request failed to connect to OpenDaylight with NorthBound REST API"
msgstr "Сбой запроса на подключение к OpenDaylight с API REST NorthBound"

#, python-format
msgid "Required field %s not specified"
msgstr "Не указано обязательное поле %s"

msgid "Resource"
msgstr "Ресурс"

msgid "Sample"
msgstr "Образец"

msgid "Samples should be included in request body"
msgstr "Образцы должны включаться в тело запроса"

#, python-format
msgid "Skip loading extension for %s"
msgstr "Пропустить загрузку расширения для %s"

#, python-format
msgid "String %s is not a valid isotime"
msgstr "Строка %s не является допустимым значением isotime"

msgid ""
"The Yaml file that defines mapping between samples and gnocchi resources/"
"metrics"
msgstr ""
"Файл Yaml, определяющий связи между образцами и ресурсами gnocchi "
"(показателями)"

#, python-format
msgid ""
"The data type %(type)s is not supported. The supported data type list is: "
"%(supported)s"
msgstr ""
"Тип данных %(type)s не поддерживается. Список поддерживаемых типов данных: "
"%(supported)s"

#, python-format
msgid "The field 'fields' is required for %s"
msgstr "Поле 'fields' является обязательным для %s"

msgid "The path for the file publisher is required"
msgstr "Требуется путь для публикатора файлов"

#, python-format
msgid "UDP: Cannot decode data sent by %s"
msgstr "UDP: не удается декодировать данные, отправленные %s"

msgid "UDP: Unable to store meter"
msgstr "UDP: не удалось сохранить счетчик"

#, python-format
msgid "Unable to connect to the database server: %(errmsg)s."
msgstr "Не удалось подключиться к серверу базы данных: %(errmsg)s."

#, python-format
msgid ""
"Unable to convert the value %(value)s to the expected data type %(type)s."
msgstr ""
"Преобразовать значение %(value)s в ожидаемый тип данных %(type)s невозможно."

#, python-format
msgid "Unable to discover resources: %s"
msgstr "Не удалось найти ресурсы: %s"

#, python-format
msgid "Unable to evaluate expression %(expr)s: %(exc)s"
msgstr "Вычислить выражение %(expr)s невозможно: %(exc)s"

#, python-format
msgid "Unable to load publisher %s"
msgstr "Не удалось загрузить публикатор %s"

#, python-format
msgid "Unable to load the hypervisor inspector: %s"
msgstr "Не удалось загрузить инспектор гипервизора: %s"

#, python-format
msgid ""
"Unable to reconnect to the primary mongodb after %(retries)d retries. Giving "
"up."
msgstr ""
"Не удалось повторно подключиться к основной базе данных mongodb после "
"%(retries)d попыток. Дальнейшие попытки прекращены."

#, python-format
msgid ""
"Unable to reconnect to the primary mongodb: %(errmsg)s. Trying again in "
"%(retry_interval)d seconds."
msgstr ""
"Не удалось повторно подключиться к основной mongodb: %(errmsg)s. Повторное "
"подключение через %(retry_interval)d секунд."

msgid "Unable to send sample over UDP"
msgstr "Не удалось отправить образец по UDP"

#, python-format
msgid ""
"Unexpected exception converting %(value)s to the expected data type %(type)s."
msgstr ""
"Возникла непредвиденная исключительная ситуация при преобразовании %(value)s "
"в ожидаемый тип данных %(type)s."

#, python-format
msgid "Unknown discovery extension: %s"
msgstr "Неизвестное расширение поиска: %s"

#, python-format
msgid "Unknown metadata type. Key (%s) will not be queryable."
msgstr "Неизвестный тип метаданных. Ключ (%s) нельзя будет запрашивать."

#, python-format
msgid ""
"Unknown status %(stat)s received on Load Balancer %(id)s, skipping sample"
msgstr ""
"В балансировщике нагрузки %(id)s получено неизвестное состояние %(stat)s, "
"пример пропускается"

#, python-format
msgid "Unknown status %(stat)s received on fw %(id)s,skipping sample"
msgstr ""
"В fw %(id)s получено неизвестное состояние %(stat)s,пример пропускается"

#, python-format
msgid "Unknown status %(stat)s received on listener %(id)s, skipping sample"
msgstr ""
"В обработчике %(id)s получено неизвестное состояние %(stat)s, пример "
"пропускается"

#, python-format
msgid "Unknown status %(stat)s received on member %(id)s, skipping sample"
msgstr ""
"В участнике %(id)s получено неизвестное состояние %(stat)s, пример "
"пропускается"

#, python-format
msgid "Unknown status %(stat)s received on pool %(id)s, skipping sample"
msgstr ""
"В пуле %(id)s получено неизвестное состояние %(stat)s,пример пропускается"

#, python-format
msgid "Unknown status %(stat)s received on vip %(id)s, skipping sample"
msgstr ""
"В vip %(id)s получено неизвестное состояние %(stat)s,пример пропускается"

#, python-format
msgid "Unknown status %(stat)s received on vpn %(id)s, skipping sample"
msgstr ""
"В VPN %(id)s получено неизвестное состояние %(stat)s, пример пропускается"

#, python-format
msgid "VM %s not found in VMware vSphere"
msgstr "Виртуальная машина %s не найдена в VMware vSphere"

#, python-format
msgid "VM %s not found in XenServer"
msgstr "Не найдена виртуальная машина %s в XenServer"

msgid "Wrong sensor type"
msgstr "Неверный тип датчика"

msgid "XenAPI not installed"
msgstr "XenAPI не установлен"

#, python-format
msgid "YAML error reading Definitions file %(file)s"
msgstr "Ошибка YAML при чтении файла определений %(file)s"

#, python-format
msgid "could not get CPU time for %(id)s: %(e)s"
msgstr "не удалось получить процессорное время для %(id)s: %(e)s"

msgid "direct option cannot be true when Gnocchi is enabled."
msgstr "Параметр direct не может быть равен true, если включен Gnocchi."

#, python-format
msgid "dropping out of time order sample: %s"
msgstr "удаление образца, выпадающего из хронологического порядка: %s"

#, python-format
msgid "dropping sample with no predecessor: %s"
msgstr "удаление образца без предшественника: %s"

msgid "ipmitool output length mismatch"
msgstr "несоответствие длины вывода ipmitool"

msgid "max_bytes and backup_count should be numbers."
msgstr "max_bytes и backup_count должны быть числами."

msgid "parse IPMI sensor data failed,No data retrieved from given input"
msgstr ""
"сбой анализа данных датчика IPMI, не получены данные из переданного ввода"

msgid "parse IPMI sensor data failed,unknown sensor type"
msgstr "сбой анализа данных датчика IPMI, неизвестный тип датчика"

msgid "running ipmitool failure"
msgstr "сбой выполнения ipmitool"
